LICENSE

```diff
@@ -0,0 +1,10 @@
+Copyright 2013 Google Inc. All Rights Reserved.
```

Comment:
oh I understand your request now. Thanks for the clarification, all updated now.